app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self._table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self._table_name}"
```

Comment:
Using f-strings for SQL queries can lead to SQL injection vulnerabilities. Consider using parameterized queries instead, which you are already doing for the values.